dali/python/nvidia/dali/plugin/mxnet.py

```diff
@@ -132,6 +136,11 @@ def __init__(self,
         self._fill_last_batch = fill_last_batch
         self._last_batch_padded = last_batch_padded
         self._auto_reset = auto_reset
+        assert self._size > 0 or (self._size < 0 and len(pipelines) == 1), "Negative size is supported only for a single pipeline"
```

Comment:
`size == 0` is not negative, but still illegal (?). Maybe there should be a separate error check for that case.

Reply:
Done